runblender.ipynb

```diff
@@ -196,28 +187,13 @@
         "with open('setgpu.py', 'w') as f:\n",
         "    f.write(data)"
       ],
-      "execution_count": 0,
-      "outputs": []
-    },
-    {
-      "cell_type": "code",
-      "metadata": {
-        "id": "QRzNmiHN8Xr0",
-        "colab_type": "code",
-        "colab": {}
-      },
-      "source": [
-        "from google.colab import drive\n",
-        "drive.mount('/gdrive')"
-      ],
-      "execution_count": 0,
+      "execution_count": 8,
```

Comment:
Could you clear the execution_count across the document and set them to 0 or null?

Reply:
Ok